src/test/java/org/fundacionjala/pivotal/testng/DeleteProjectTest.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionjala.pivotal.testng;
+
+import org.fundacionjala.core.util.Environment;
+import org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard;
+import org.fundacionjala.pivotal.pageobjects.projects.Projects;
+import org.fundacionjala.pivotal.pageobjects.login.SignInPage;
+import org.fundacionjala.core.ui.WebDriverManager;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+/**
+ * @author Adrian Rojas - AWT-[01].
+ * @version 0.1
+ */
+public class DeleteProjectTest {
+    private Dashboard dashboard;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "dashboard" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)